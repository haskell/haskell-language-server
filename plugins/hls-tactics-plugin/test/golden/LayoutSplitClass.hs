class Test a where
  test :: Bool -> a
  test x = _

