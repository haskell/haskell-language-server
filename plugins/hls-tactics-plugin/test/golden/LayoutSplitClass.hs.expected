class Test a where
  test :: Bool -> a
  test False = _0
  test True = _1

